platforms/core-execution/workers/src/integTest/groovy/org/gradle/workers/internal/WorkerDaemonProcessFailureIntegrationTest.groovy

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.workers.internal
+
+import org.gradle.integtests.fixtures.ProcessFixture
+import org.gradle.internal.os.OperatingSystem
+import org.gradle.test.fixtures.file.TestFile
+import org.gradle.test.fixtures.server.http.BlockingHttpServer
+import org.gradle.test.precondition.Requires
+import org.gradle.test.preconditions.UnitTestPreconditions
+import org.junit.Rule
+
+@Requires(UnitTestPreconditions.Jdk9OrLater)
+class WorkerDaemonProcessFailureIntegrationTest extends AbstractDaemonWorkerExecutorIntegrationSpec {
+    @Rule
+    final BlockingHttpServer blockingHttpServer = new BlockingHttpServer()
+    boolean pidProcessorProjectGenerated
+
+    def setup() {
+        blockingHttpServer.start()
+        if (OperatingSystem.current().windows) {
+            executer.withStackTraceChecksDisabled()
```

Comment:
Can we document why stack traces are disabled for windows?

Reply:
Done.